pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -40,6 +41,10 @@ assert x11Support -> tcl != null
                   && tk != null
                   && xorgproto != null
                   && libX11 != null;
+
+assert bluezSupport -> sourceVersion.major == "3"
```

Comment:
this expression is only for Python 3

Reply:
What kind of change are you requesting?

My intention was to enforce Python 3 for BlueZ support because I do not see Bluetooth related sockets in Python 2, https://docs.python.org/2/library/socket.html.

Reply:
I understand, but this expression is never called for Python 2. Python 2 has its own expression, thus there is no need to check major version.

Reply:
Oh.. you're totally right; thank. I removed the redundant check.

Would you like to take another look at the proposed changes?